draft-ietf-acme-acme.md

```diff
@@ -1544,16 +1556,10 @@ Content-Type: application/jose+json
 }
 ~~~~~~~~~~
 
-Revocation requests are different from other ACME request in that they can be
-signed either with an account key pair or the key pair in the certificate.
 Before revoking a certificate, the server MUST verify that the key used to sign
 the request is authorized to revoke the certificate.  The server SHOULD consider
-at least the following keys authorized for a given certificate:
-
-* the public key in the certificate.
-
-* an account key that is authorized to act for all of the identifier(s) in the
-  certificate.
+authorized any account key that is authorized to act for all of the identifier(s) in the
```

Comment:
This reads a little bit awkwardly to me. What about:

"The server SHOULD consider any account key authorized for all of the identifier(s) in the certificate to be authorized to revoke the certificate." ?